### PR TITLE
fix #78527, #78652

### LINKED ISSUE #78527: elazigonline.com
### Issue URL (Ads)
 [https://elazigonline.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Felazigonline.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish

### PR DESCRIPTION
**Pull request by wout-codes**
#78527, #78652

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -2555,7 +2555,7 @@ sciencechannel.com,go.discovery.com,foodnetwork.com,tlc.com##.videoMetadata__soc
 focus.pl##.article-social-media
 innpoland.pl,natemat.pl,mamadu.pl##.fb_share_button
 intersinema.com##.shareBig
-cryptodaily.co.uk,elazighaber23.com,kayisihaber.com,wowkeren.com,gercekfethiye.com,gunisigigazetesi.net##.post-share-area
+elazigonline.com,cryptodaily.co.uk,elazighaber23.com,kayisihaber.com,wowkeren.com,gercekfethiye.com,gunisigigazetesi.net##.post-share-area
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/specific.txt:2558]: this rule blocks comment section in articles

Comment (Sergey-Lyapin) [reply SocialFilter/sections/specific.txt:2558]: these share buttons are missed
share

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: SocialFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply SocialFilter/sections/specific.txt:2558]: Fixed.

Comment (wout-codes) [reply SocialFilter/sections/specific.txt]: I have also added another fix for a similar issue.

File: SocialFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 !
 ! Extended CSS rules
 !
+kentteyasam.com,elazigonline.com#?#.si-share:upward(3)
```
Comment (Sergey-Lyapin) [inline SocialFilter/sections/css_extended.txt:4]: We can avoid ext-css:
```
kentteyasam.com,elazigonline.com##div[class^="nws-share-"]
kentteyasam.com,elazigonline.com##.si-share
kentteyasam.com,elazigonline.com##.post-share-area:not(.pb-0)
```

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Pull request merged**